quiz/models.py

```diff
@@ -43,7 +43,7 @@ def __str__(self):
 class Result(models.Model):
```

Comment:
The name 'Result' is rather confusing leading you to think that this is the model that logs your score results for each test you submit. You can rename it to something like 'ScoreRange' or something else.